docs/getting-started/ingest-data.asciidoc

```diff
@@ -39,7 +42,7 @@ ships these data sources:
 [[install-beats]]
 == Install {beats} shippers
 
-To populate {es-sec} with hosts and network security events, you need to install and
+To populate {elastic-sec} with hosts and network security events, you need to install and
```

Comment:
```suggestion
To populate {elastic-sec} with host and network security events, you need to install and
```
not sure about this one - as written it seems to mean "populate with hosts, and also with network security events". My change is intended to make it mean "populate with security events related to the host or the network".

Reply:
I think it's meant to be populating with "hosts" and also with "network security events." Another possible revision?
```suggestion
To populate {elastic-sec} with network security events from hosts, you need to install and
```

Reply:
I think it's important to keep the "hosts" bit separate. If I'm understanding the sentence correctly, it's telling users that they can add and monitor hosts through the security app and stream network security events to the Security app. @jmikell821 WDTY?

Reply:
Inserting the following as discussed in Slack:
To add hosts and populate {elastic-sec} with network security events.

Reply:
Adding the following as per group discussion on Slack:
To add hosts and populate {elastic-sec} with network security events.